packages/invoices/src/OutboundShipment/api/api.ts

```diff
@@ -47,7 +48,7 @@ const outboundParsers = {
       }
     }
   },
-  toStatus: (patch: RecordPatch<Invoice>) => {
```

Comment:
also removed the use of the `Invoice` type (previously had removed most uses) as it has the same problems as the `InvoiceLine` type